.github/workflows/codestyle.yml

```
@@ -25,33 +27,39 @@ jobs:
 
     strategy:
       matrix:
-        php: [8.3]
-        dependency-version: [prefer-stable]
+          php: [8.1]
 
     name: Codestyle check
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
           php-version: ${{ matrix.php }}
           coverage: none
-          tools: composer:v2
+          tools: composer:v2, cs2pr
 
       - name: Cache Composer packages
         id: composer-cache
         uses: actions/cache@v3
         with:
-          path: vendor
-          key: ${{ runner.os }}-php-${{ matrix.php }}-${{ matrix.dependency-version }}-${{ hashFiles('**/composer.json') }}
+          path: |
+            vendor
+            .php-cs-fixer.cache
+          key: ${{ runner.os }}-codestyle-${{ matrix.php }}-${{ hashFiles('**/composer.json') }}
           restore-keys: |
-            ${{ runner.os }}-php-${{ matrix.php }}-${{ matrix.dependency-version }}
+            ${{ runner.os }}-codestyle
 
       - name: Install dependencies
         run: composer install --prefer-dist --no-progress
 
       - name: Run PHPCS for the codestyle
         run: vendor/bin/phpcs
+
+      - name: Run PHP Coding Standards Fixer
+        run: |
+          vendor/bin/php-cs-fixer fix --dry-run --diff --ansi || true
+          vendor/bin/php-cs-fixer fix --dry-run --format=checkstyle | cs2pr
```

Does this auto-fix and commit through GitHub action?

If so, I'd prefer to just get notified of erros.

If the code gets autof-fixed people just get lazy and don't care and every commit of them will get auto-fixed which leads to many commit from GitHub.

---

> Does this auto-fix and commit through GitHub action?

Absolutely not, that would be terrible (like we talked about). Gives a nice diff in the log and notifications in each of the affected files.

Hench the `--dry-run` argument.

---

Ah, OK. Sorry, overlooked the `dry-run` option.
Thanks for clarifying.

Let's clarify the step name, then:

```suggestion
      - name: Run PHP Coding Standards Fixer Check
        run: |
          vendor/bin/php-cs-fixer fix --dry-run --diff --ansi || true
          vendor/bin/php-cs-fixer fix --dry-run --format=checkstyle | cs2pr
```